src/components/PersonLink.tsx

```diff
@@ -0,0 +1,72 @@
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import React from 'react';
+import { Person } from '../types';
+import cn from 'classnames';
+
+type Props = {
+  person: Person;
+  searchPeople: (name: string | null) => Person | undefined;
+};
+
+export const PersonLink: React.FC<Props> = ({ person, searchPeople }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const mother = searchPeople(person.motherName);
+  const father = searchPeople(person.fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={{
+            pathname: `../${person.slug}`,
```

Comment:
The `pathname` property should be a string, and the use of `../${person.slug}` assumes a relative path. Ensure that this path resolves correctly in your application's routing structure. If the path is incorrect, it may lead to navigation issues.